src/lib.ts

```diff
@@ -140,68 +141,27 @@ export const args = Args.create("crimbo23", "A script for farming elf stuff", {
     help: "The number of turns to run (use negative numbers for the number of turns remaining)",
     default: Infinity,
   }),
-  zone: Args.string({
-    options: [
-      ["armory", "Armory"],
-      ["bar", "The Bar"],
-      ["cafe", "Cafe"],
-      ["cottage", "Abuela's Cottage"],
-      ["foundry", "Pirate Foundry"],
-    ],
-    default: "cottage",
-  }),
-  affiliation: Args.string({
-    options: [
-      ["none", "Do not pick a side"],
-      ["elves", "Fight for the elves"],
-      ["pirates", "Fight for the Crimbuccaneers"],
-    ],
-    help: "The side to fight for.",
-    default: "elves",
-  }),
+  island: Args.custom<Island[]>({
+    hidden: false,
+  }, (str) => {
+    const splitStr = str.split(",");
+    if (![1, 2].includes(splitStr.length)) return new ParseError("Can only select 1 or 2 islands!");
+    if (!CrystalBall.have() && splitStr.length === 2) return new ParseError("Can only select 2 islands with orb!");
```

Comment:
```suggestion
    if (![1, 2].includes(splitStr.length)) return new ParseError("Please select at least 1 island, and at most 2");
    if (!CrystalBall.have() && splitStr.length === 2) return new ParseError("Without miniature crystal ball, you may only select a single island.");
```

Hopefully more clarity in error messaging